extensions/adobe/experience/customerJourneyManagement/engageable-profile-mixins.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/engageableProfile",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Engageable Profile Fields",
+  "description": "A set of meta-data fields related to engageable profile.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "definitions": {
+    "entities": {
+      "properties": {
+        "https://ns.adobe.com/experience/entities/engagement": {
+          "title": "Engagement Fields",
+          "type": "object",
+          "description": "Fields defined for engagement",
+          "properties": {
```

Comment:
Not sure why you need this additional `solution` object inside object `engagement` object. Do you plan to have more fields under `solution` object in the future?

Reply:
@anandphatak not right now but since we visualised this to be used by different products as well kept this as object so that solution specific other field could be stored, do you see any challenge with this?